cmd/healthcheck.go

```diff
@@ -0,0 +1,38 @@
+package cmd
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/miekg/dns"
+	"github.com/spf13/cobra"
+)
+
+const (
+	defaultDNSPort = 53
+)
+
+func NewHealthcheckCommand() *cobra.Command {
+	c := &cobra.Command{
+		Use:   "healthcheck",
+		Short: "performs healthcheck",
+		RunE:  healthcheck,
+	}
+
+	c.Flags().Uint16P("port", "p", defaultDNSPort, "healthcheck port 5333")
```

Comment:
should it be port 53?

Reply:
No as this is the default value. Thought the description should be some value different from that.